tests/components/tplink/fixtures/features.json

```diff
@@ -44,6 +44,26 @@
     "type": "Switch",
     "category": "Config"
   },
+  "motion_detection": {
+    "value": true,
+    "type": "Switch",
+    "category": "Primary"
```

Comment:
These should rather be config, no?

Reply:
These are primary features in the lib so should be the same in the fixtures.

Reply:
Maybe we should have a clearer definition for what primary controls for a device are, but in my eyes controls that do not cause immediate state change on the device (like, e.g., brightness on bulbs does) would belong to the configuration category.

In homeassistant, this affects what's shown on  the default dashboard, and I would not personally expect to see these controls on there as they are not something you need to access on a daily basis.

Reply:
We can always move these later as it has no breaking effect on HA, but that’s a lib change not affecting this PR.

Reply:
Indeed, just wanted to mention that we, IMO, should change the upstream categorization, if that's the case.